### DISCUSSION TITLE
Help me please

### DISCUSSION THREAD

**Discussion by RyanS22604** [GriefPrevention/GriefPrevention#1386] [Q&A]
Please I need help!!!!
Is there anyway to make it so that they can only claim land in certain multiverse worlds or at least make it so that they arent allowed to claim in certain areas!!??

**Comment by bigpresh** [comment]
If you read the [Setup and Configuration](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration) guide, the first section is "Limiting Which Worlds GriefPrevention Runs On" which tells you how to use the `Claims.mode` setting to control GP per-world.
